domain/model/pokemon.go

```diff
@@ -0,0 +1,44 @@
+package model
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	fileName string = "pokemons.csv"
+)
+
+type Pokemon struct {
+	ID   int
+	Name string
+	Url  string
+}
+
```

Comment:
I don't think the model layer has the responsibility of handle the logic for the 3 functions below.
Maybe the use case layer?

Reply:
Agreed, I moved the methods and delete some of them. I don't currently use the toString method but I leave it as a "extension method" (receiver method) but outside the struct itself in a new file for all posible extension methods, this way is available for other interactions  **e.g. on logs** otherwise if I leave it as a method in the use case layer (presenter) won't be available application wide. Please let me know if this approach is acceptable.